tests/performance/common/common.go

```diff
@@ -199,3 +207,88 @@ func ApplyLabel(nodeHandler string) error {
 	}
 	return nil
 }
+
+// AddFakePod adds a fake pod
+func AddFakePod(ControllerHubURL string, pod types.FakePod) {
+	reqBody, err := json.Marshal(pod)
+	if err != nil {
+		log.LOGGER.Errorf("Unmarshal HTTP Response has failed: %v", err)
```

Comment:
I think we don't use beehive's log package in perf/e2e tests. 
cc @pavan187

Reply:
Thanks. I will update later